crates/rlbot/examples/atba_raw/main.rs

```diff
@@ -33,15 +33,15 @@ fn main() {
         let packet = rlbot_connection.recv_packet().unwrap();
         if let Packet::ControllableTeamInfo(x) = packet {
             break x;
-        } else {
-            packets_to_process.push(packet);
-            continue;
         }
+
+        packets_to_process.push(packet);
```

Comment:
unless this is clippy, I think it was better before the change

Reply:
yeah, clippy lint. you _dont_ need the else after a break/return/continue, and the continue at the end of the statement was useless too

Reply:
imo, its more clear what it does before clippy but its not a big deal